charts/gatekeeper-artifacts/templates/constraints.yaml

```diff
@@ -0,0 +1,6 @@
+{{- with .Values.constraintFiles }}
+{{- range $file := . }}
+    {{- $file | toYaml }}
+---
+{{- end }}
+{{- end }}
```

Comment:
The `/crds` folder of this chart should be empty and have a `.gitignore` file with `*`, as the prehook should generate the crds and put them there.

Reply:
yes indeed. But these are the CR resources